nunit.xamarin.sln

```diff
@@ -0,0 +1,356 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.27428.2005
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{DABD2667-0EF4-4814-8353-8675E1C8F95E}"
+	ProjectSection(SolutionItems) = preProject
+		.gitattributes = .gitattributes
+		.gitignore = .gitignore
+		build.cake = build.cake
+		README.md = README.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Src", "Src", "{B3AE763A-DC9C-4F5D-B5F0-97A1B10D147B}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{040381B1-C9E8-4D7C-8A60-EF4C343C7374}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "NuGet", "NuGet", "{9535215D-5DAC-4E52-A0F2-5216F933234B}"
+	ProjectSection(SolutionItems) = preProject
+		nuget\nunit.runners.xamarin.nuspec = nuget\nunit.runners.xamarin.nuspec
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "MonoAndroid10", "MonoAndroid10", "{82E94D4E-EB38-4D00-A900-EA83CB465DEA}"
+	ProjectSection(SolutionItems) = preProject
+		nuget\MonoAndroid10\MainActivity.cs.txt.pp = nuget\MonoAndroid10\MainActivity.cs.txt.pp
+	EndProjectSection
+EndProject
```

Comment:
All these files are removed, aren't they?

Reply:
Missed this. Will fix.